documentation/src/main/asciidoc/public/reference/_search-dsl-predicate.adoc

```diff
@@ -1572,6 +1590,176 @@ but can be <<search-dsl-predicate-common-constantScore,made constant with `.cons
+with disabled position increments, phrase query `"book shelve"` will not match such document.
+
+.Matching a query string: disabling position increments
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java[tags=queryString-enable-position-increments]
+----
+====
+
+[[search-dsl-predicate-query-string-rewrite-method]]
+=== Rewrite method
+
+The rewrite method determines how the backend's query parser rewrites and scores multi-term queries.
+To change the default `CONSTANT_SCORE` rewrite method, one of the allowed `RewriteMethod` enum values can be passed
+to `.rewriteMethod(RewriteMethod)`/`rewriteMethod(RewriteMethod, int)`.
+We will not go into details about different rewrite methods in this guide.
+To learn more about them, please refer to your backend's guide
+(link:{elasticsearchDocUrl}/query-dsl-multi-term-rewrite.html[Elasticsearch]/link:{openSearchDocUrl}/query-dsl/full-text/query-string/[OpenSearch]/link:{luceneJavadocUrl}org/apache/lucene/search/MultiTermQuery.html[Lucene]).
+
+.Matching a query string: rewrite method
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java[tags=queryString-rewrite-method]
+----
+====
+
+[[search-dsl-predicate-query-string-minimumshouldmatch]]
+=== `minimumShouldMatch`: fine-tuning how many `should` clauses are required to match
+
+The resulting query parsed from the query string may result in a boolean query with should clauses.
+It may be helpful to control how many `should` clauses must match to consider a document as a match.
+
+.Fine-tuning `should` clauses matching requirements with `minimumShouldMatch`
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java[tags=simpleQueryString-minimum-should-match]
+----
+====
+
+This is similar to <<search-dsl-predicate-boolean-minimumshouldmatch,boolean>> or
+<<search-dsl-predicate-simple-query-string-minimumshouldmatch,simple query string>>
+predicates `minimumShouldMatch` options.
+
+[[search-dsl-predicate-query-string-multiple-fields]]
+=== Targeting multiple fields
+
+Optionally, the predicate can target multiple fields.
+In that case, the predicate will match documents for which _any_ of the given fields matches.
+
+See <<search-dsl-predicate-common-multiple-fields>>.
+
+IMPORTANT: If targeted fields have different analyzers, an exception will be thrown.
+You can avoid this by <<search-dsl-predicate-common-overriding-analysis,picking an analyzer explicitly>>,
+but make sure you know what you're doing.
+
+[[search-dsl-predicate-query-string-other]]
+=== Other options
+
+* The score of a `queryString` predicate is variable by default,
+but can be <<search-dsl-predicate-common-constantScore,made constant with `.constantScore()`>>.
```

Comment:
You said above that the default rewrite method was "CONSTANT_SCORE"... might want to explain that this doesn't mean, in fact, that the score is constant...